dist/wrapper.js

```diff
@@ -77,6 +75,8 @@ var zenroom = function () {
    * @example <caption>Example usage of `script()`</caption>
    * // returns zenroom
    * import zenroom from 'zenroom'
+   * // or without ES6 syntax
+   * // const zenroom = require('zenroom').default
```

Comment:
can you please update also the README.md?

Reply:
The .wasm and .js files are the output from [Zenroom @ afb3ce8](https://github.com/DECODEproject/zenroom/tree/afb3ce8bb23063b007586e8d1f70d6864fb87d91), the commit referred from the submodule, and emscripten 

Actually, compiling from the master branch of Zenroom, the wasm produced won't pass the tests in which the script contains a reference to the global `ECDH` and spits out these messages:
```
[string "..."]:2: attempt to index a nil value (global 'ECDH')
[string "ZEN:begin(1)..."]:1: attempt to index a nil value (global 'ZEN')[!]
```
I'd ask @jaromil to give me a hint on how to address this problem

Reply:
is it fixed now?